.github/scripts/auto-backport.py

```diff
@@ -97,7 +97,7 @@ def backport(repo, pr, version, commits, backport_base_branch, user):
             repo_local.git.checkout(b=new_branch_name)
             try:
                 fork_repo = pr.user.get_repo(repo.name)
-            except Exception as e:
```

Comment:
I think it would be better to also keep the general exception instead of letting it fly.
Maybe with another printing:
```suggestion
 except Exception as ge:
    print(f"Got general error: {ge}")
```

Reply:
please note, `get_repo()` does not throw `Exception`. if it does, then it's an *exception*. and we should let it fly.